Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -32,6 +32,9 @@ private let commentLinePrefixCharacterSet: CharacterSet = {
     return characterSet
 }()
 
+// https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/LexicalStructure.html#//apple_ref/swift/grammar/line-break
```

Comment:
CI is failing because this line is too long. Should I remove it or just disable the violation?